### ISSUE TITLE
Consider schema.org as data model

### ISSUE THREAD

**Issue by rajido** [codemeta/codemeta#103]
Schema.org could be the data model used by codeMeta. This would be an incentive for adoption since codeMeta entries annotated with Scehma.org would be indexed by search engines (better visibility of our software). Schema.org codeMeta related types are Code, SoftwareSourceCode and SoftwareApplication. These schema.org types are not very well defined. codeMeta has the chance to improve them via the good work done so far by mapping resources in the [crosswalk file](https://github.com/codemeta/codemeta/blob/master/crosswalk.csv). This would be easier once we  include Schema.org to the [crosswalk](https://github.com/codemeta/codemeta/blob/master/crosswalk.csv) mapping table #101.  Schema.org allows the use of non schema.org vocabularies so the codeMeta model could be used straight away if we decided to use schema.org as skeleton. Schema.org also recommends 3 format for implementation: RDFa, JSON-LD and Microdata and has build a lot of tools that we can reuse. e.g. validators, crawlers, etc.

**Comment by mbjones** [comment]
Thanks, @rajido. Schema.org is already the base data model used by CodeMeta.  The alignment with schema.org is something we've discussed since the start.  So much so that we made the decision to base the whole CodeMeta schema on schema.org types whenever they exist.  So, we are implicitly doing a mapping already, in the sense that, whenever an appropriate schema.org field exists, we use that as the type of the corresponding field in CodeMeta.  It would be fine to add another column to the crosswalk to show these mappings explicitly, so I'll keep #101 open as well.

The namespaces that we are drawing from are, in priority order: schema.org, dcterms, and xsd.  Anything not in those namespaces has been placed in a codemeta namespace.

Over the long run, it would be wonderful to get the CodeMeta refinements and extensions incorporated directly into schema.org, but I think that is a bit of a longer-term project for our community.  So, I will leave this ticket open as the kernel of future discussions on how to make that happen.  In the meantime, I'd like to get a released version of the codemeta.jsonld file so that our communities can start to use it and gain experience in what works and what needs improvement.
